events/views.py

```diff
@@ -36,3 +37,24 @@ def create_event(request):
     else: # If logged in user is accessing the page through a normal GET request
         form = EventForm()
         return render(request, 'create_event.html',{'form':form})
+
+
+def edit_event(request, id):
+
+    try:
+        event = Event.objects.get(id=id)
+
```

Comment:
if event.author.id != request.user.id:
    return 404('User does not match Author')

We'll modify later if this is not secure.
If this is the only change, we'll just pull. No need to make a new commit for a small change